c++/src/kj/test.h

```diff
@@ -128,6 +141,17 @@ bool expectFatalThrow(Maybe<Exception::Type> type, Maybe<StringPtr> message,
 // fork() is not available, this always returns true.
 #endif
 
+bool expectExit(Maybe<int> statusCode, Function<void()> code);
```

Comment:
It looks like the first parameter to both of these functions can just be `int` instead of `Maybe<int>` since the corresponding macros never pass `nullptr`?

Reply:
You can pass `nullptr` in manually in case where you don't know the exit code or signal for sure. I've added couple tests with nullptr too.